arch.mk

```diff
@@ -139,6 +139,12 @@ ifeq ($(ARCH),ARM)
     ARCH_FLASH_OFFSET=0x08000000
   endif
 
+  ifeq ($(TARGET),stm32f103)
```

Comment:
I think we should rename the target (and the associated files) to `stm32f1` for consistency with the other ports, to cover the entire family.

Comments in the files, referring to STM32F103, are OK, and even mentioning the BluePill in the test-app as you did is actually helpful as reference if anyone is testing on a different STM32F1.

Reply:
Sure!